lab1.js

```diff
@@ -104,7 +121,11 @@ var nextAnimal;
 // TODO: 12 points
 // Assign one of your favorite animals to nextAnimal using Math.random() to pick
 
-// your code goes here
+function pickRandom(min, max) {
+  return Math.floor(Math.random() * (max - min + 1)) + min;
```

Comment:
nice. Can also pass in an array, and use that array's length instead of the max/min logic.